targets/TARGET_ARM_SSG/TARGET_MUSCA_A1/TARGET_MUSCA_A1_S/device/TOOLCHAIN_ARMC6/musca_s.sct

```diff
@@ -0,0 +1,151 @@
+#! armclang --target=arm-arm-none-eabi -march=armv8-m.main -E -xc
+
+/*
+ * Copyright (c) 2018 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "../../../partition/region_defs.h"
+
+#if !defined(TFM_LVL)
+  #define TFM_LVL   1
+#endif
+
+#if !defined(MBED_ROM_START)
+  #define MBED_ROM_START    S_CODE_START    // 0x10020400
+#endif
+
+#if !defined(MBED_ROM_SIZE)
+  #define MBED_ROM_SIZE     IMAGE_CODE_SIZE // 0x7f800
+#endif
+
+#if !defined(MBED_RAM_START)
+  #define MBED_RAM_START    S_DATA_START    // 0x30000000
+#endif
+
+#if !defined(MBED_RAM_SIZE)
+  #define MBED_RAM_SIZE     S_DATA_SIZE     // 0x10000
+#endif
+
+LR_CODE MBED_ROM_START MBED_ROM_SIZE {
+
+    /****  This initial section contains common code for TEE */
+    ER_TFM_CODE MBED_ROM_START S_CODE_SIZE {
+        *.o (RESET +First)
+        .ANY (+RO)
+    }
+
+#if TFM_LVL == 1
```

Comment:
Do we need this check given the non-secure configuration is using a different linker script?

Reply:
The TFM level doesn't care about the non-secure side, only the partitioning level in the secure side.